src/build.d

```diff
@@ -634,6 +679,8 @@ void parseEnvironment()
     env.getDefault("GIT_HOME", "https://github.com/dlang");
     env.getDefault("SYSCONFDIR", "/etc");
     env.getDefault("TMP", tempDir);
+    env.getDefault("TOOLS_DIR", "../../tools");
```

Comment:
Can we base the default on the directory of this script and not depend on the current working directory?

Reply:
Done